certify-service/src/main/resources/application-local.properties

```diff
@@ -241,6 +238,13 @@ mosip.certify.integration.scan-base-package=io.mosip.certify.mock.integration
 mosip.certify.integration.audit-plugin=LoggerAuditService
 mosip.certify.integration.vci-plugin=MockVCIssuancePlugin
 mosip.certify.mock.vciplugin.verification-method=${mosip.certify.authn.jwk-set-uri}
+#mosip.certify.mock.authenticator.get-identity-url="https://api.dev1.mosip.net/v1/mock-identity-system/identity"
+mosip.certify.mock.authenticator.get-identity-url="http://localhost:8082/v1/mock-identity-system/identity"
+mosip.certify.mock.vciplugin.p12.filename=mock-issuer-details.p12
+#TODO: get the secret for password onboarded
+mosip.certify.mock.vciplugin.p12.password=${mosip.certify.mock.vciplugin.keystore.password}
```

Comment:
Please let me know if any name changes are required for the secret "mosip.certify.mock.vciplugin.keystore.password"